### ISSUE TITLE
unreproducible routing hell

### ISSUE THREAD

**Issue by 532910** [osmandapp/Osmand#9545]
Sometimes OsmAnd routes unsuitable ways.

There are no car road, only pedestrian can cross railway here: https://osm.org/node/6937749008
![image](https://user-images.githubusercontent.com/14617699/89005813-0bd91380-d30e-11ea-891a-b6069b0718cd.png)

Some time ago it offered to drive a car by railroad. (No screenshot, just believe.)

**Comment by vshcherb** [comment]
Not reproducible, we need to investigate more

**Comment by 532910** [comment]
The route shown on the screenshot above is valid for pedesrians. Could this somehow be mixed?

**Comment by vshcherb** [comment]
Yes we've heard about those issues when user switches between profiles fast, so route could remain from previous profile. Though we've never heard about it when user doesn't change profile to pedestrian

**Comment by 532910** [comment]
I'm sure I not switched between profiles that time.

**Comment by 532910** [comment]
And again it routed me as pedestrian, while car was selected.
I tried to kill osmand and restart it but this doesn't help.
